doc/cephfs/fs-volumes.rst

```diff
@@ -1340,5 +1340,28 @@ set with this id was present in the database
 
   $ ceph fs quiesce fs1 sub1 sub2 sub3 --set-id="external-id" --if-version=0
 
+
+.. _disabling-volumes-plugin:
+
+Disabling Volumes Plugin
+------------------------
+By default, on a Ceph cluster, volumes plugin is enabled as well as set as
+``always on`` module. However, in certain cases it might be necessary/favourable
+to disable it. For example, when a CephFS is in a degraded state, the volumes
+plugin commands may get accumulated in MGR instead of getting served. Which
+eventually causes policy throttles to kick in and MGR becomes unresponsive.
+
+For such cases, volumes plugin can be disabled even though it is an
+``always on`` module in MGR. To do so, run ``ceph mgr module disable volumes
+--yes-i-really-mean-it``. Do note that this command will disable operations
+and remove commands of volumes plugin since it will disable all CephFS
```

Comment:
s/remove//
of the ``volumes``

Reply:
Better to keep "remove" along with "disable" since that's similar error message will be seen by users on using these commands after disabling the module.